tests/push/test_manager.py

```diff
@@ -90,6 +90,28 @@ def new_start(*args, **kwargs):  # pylint: disable=unused-argument
         assert feedback_loop.get() == Status.PUSH_RETRYABLE_ERROR
         assert timer_mock.mock_calls == [mocker.call(0, Any())]
 
+    def test_empty_auth_respnse(self, mocker):
```

Comment:
correct me if i'm wrong but this test seems to be equal to the one below, moreover you have typos

Reply:
how about we update that function that fetches the token to either return a token (regardless of whether push is enabled or not) or throw an exception otherwise? That will help keep the API cleaner